internal/services/containers/container_group_resource_test.go

```diff
@@ -1371,7 +1371,7 @@ resource "azurerm_container_group" "test" {
 
   dns_config {
     nameservers    = ["reddog.microsoft.com", "somecompany.somedomain"]
-    options        = ["one:option", "two:option", "red:option", "blue:option"]
+    options        = null
     search_domains = ["default.svc.cluster.local."]
```

Comment:
Since both of these fields are optional I would test omitting them (equivalent to setting them to null) here to confirm that not setting them doesn't cause a diff in the plan like it did previously 
```suggestion
```